### PR TITLE
Add focus shortcuts

### LINKED ISSUE #4241: A11Y: Implement navigational (focus) shortcuts
## Feature Description

We start with 3 simple shortcuts for moving between these three sections:

1. <kbd>cmd/win</kbd>+<kbd>opt/alt</kbd>+<kbd>1</kbd> Currently active library tab
1. <kbd>cmd/win</kbd>+<kbd>opt/alt</kbd>+<kbd>2</kbd> Currently selected element(s)
1. <kbd>cmd/win</kbd>+<kbd>opt/alt</kbd>+<kbd>3</kbd> Currently active inspector tab

## Alternatives Considered

* <kbd>cmd</kbd>+<kbd><</kbd>/<kbd>></kbd> for cycling between the three sections, but no clear notation is possible due to different keyboard layouts.

## Additional Context

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by barklund**
## Summary

This adds focus shortcuts to the three main sections of the editor as per #4241.

## Relevant Technical Choices

* Had to modify some things in the global keyboard shortcut system, but everything else seems to work still.
* Also had to make minor changes to how the keyboard outlines work (digits can also make them appear).

## Testing Instructions

---

Fixes #4241

### CODE REVIEW

File: assets/src/edit-story/components/canvas/karma/selection.karma.js [hunk 1]
```diff
@@ -89,4 +89,19 @@ describe('Selection integration', () => {
...
+    // Move to the canvas selection.
+
+    // Click elsewhere
+    await fixture.events.click(fixture.editor.canvas.header.title);
+    expect(fixture.editor.canvas.header.title).toHaveFocus();
+
+    // Return focus with shortcut
+    await fixture.events.keyboard.shortcut('mod+alt+2');
+    expect(fixture.editor.canvas.header.title).not.toHaveFocus();
+    await fixture.snapshot('selected element has focus');
```
Comment (barklund) [inline assets/src/edit-story/components/canvas/karma/selection.karma.js:105]: I know this isn't the _"strongest"_ test ever, but validating that focus is actually on the frame (which it eventually will be) is pretty flaky and works only _sometimes_. We could try to use arrow keys and validate, that the on-canvas object moves, which indicates that it has some kind of focus (though that's also a global listener, actually).

That's actually the key issue here. Most listeners on the selected element(s) are global listeners anyway, so it's not _that_ important that the specific element has focus - it's more important that the input field **doesn't**.

File: assets/src/edit-story/components/keyboard/index.js [hunk 2]
```diff
@@ -53,7 +53,7 @@ const globalRef = createRef();
 
 function setGlobalRef() {
   if (!globalRef.current) {
-    globalRef.current = document;
+    globalRef.current = document.documentElement;
```
Comment (barklund) [inline assets/src/edit-story/components/keyboard/index.js:56]: @dvoytenko, I noticed that some events did not bubble all the way to the `document`, but only to the `documentElement`, so I changed this line here. Can you think of any drawbacks to this?

Comment (barklund) [reply assets/src/edit-story/components/keyboard/index.js:56]: In particular, a `Cmd+Option+1` shortcut pressed inside an input element only bubbled to the `documentElement`, not the `document`. However, if the same combo was pressed on a button element, it bubbled all the way to the `document`. I have no idea why!

File: assets/src/edit-story/components/inspector/karma/inspectorTabs.karma.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+      expect(documentTab).toHaveFocus();
+
+      // Click elsewhere
+      await fixture.events.click(fixture.editor.canvas.header.title);
+      expect(documentTab).not.toHaveFocus();
+
+      // Return focus with shortcut
+      await fixture.events.keyboard.shortcut('mod+alt+3');
+      expect(documentTab).toHaveFocus();
+      await fixture.snapshot('document tab has focus');
```
Comment (barklund) [inline assets/src/edit-story/components/inspector/karma/inspectorTabs.karma.js:55]: The snapshot doesn't actually contain the focus outline, even though it shows up when running karma tests locally. Not sure why.

File: assets/src/edit-story/components/library/karma/libraryTabs.karma.js [hunk 4]
```diff
@@ -85,5 +90,23 @@ describe('LibraryTabs integration', () => {
...
+      expect(textTab).toHaveFocus();
+
+      // Click elsewhere
+      await fixture.events.click(fixture.editor.canvas.header.title);
+      expect(textTab).not.toHaveFocus();
+
+      // Return focus with shortcut
+      await fixture.events.keyboard.shortcut('mod+alt+1');
+      expect(textTab).toHaveFocus();
+      await fixture.snapshot('text tab has focus');
```
Comment (barklund) [inline assets/src/edit-story/components/library/karma/libraryTabs.karma.js:109]: The snapshot doesn't actually contain the focus outline, even though it shows up when running karma tests locally. Not sure why.

**Review by miina — APPROVED** [review]
LGTM other than the inconsistency with the `+2` when there is no element selected -- sometimes the focus still goes to canvas (even though it's not clear what exactly has the focus), sometimes not.

Ok by me to leave addressing this for a separate PR.

**Pull request merged**